packages/xo-server/src/xo-mixins/backups-ng/index.mjs

```diff
@@ -481,7 +481,6 @@ export default class BackupNg {
             result = handleBackupLog(log, {
               logger,
               localTaskIds,
-              rootTaskId,
```

Comment:
You're fix is correct, but I'm really thinking we should add some documentation on `handleBackupLog` params, it's very difficult to understand how they behave :slightly_frowning_face: 

I'll try to do a basic PR on it soon.

Reply:
yes especially the rootTaskId that is set to runJobIb that changes completly the behaviour

Reply:
Feel free to add documentation (even if basic) when you encounter such code.